app/src/components/notification/context-menu/Notification-context-menu.tsx

```diff
@@ -1,8 +1,72 @@
 import React from "react";
 import "./notification-context-menu-style.css";
+import TrashIcon from "../../../images/trash-icon.svg";
+import MuteIcon from "../../../images/mute-icon.svg";
+import TurnOffIcon from "../../../images/turn-off-icon.svg";
+import { INotificationCardData } from "../notification.types";
 
-function NotificationContextMenu() {
-  return <div className="Context-menu__main-body">Place holder</div>;
+export enum NotificationContextMenuAction {
+  DELETE = "delete",
+  MUTE = "mute",
+  TURN_OFF = "turn_off",
+}
+
+const ToolTip = ({ text }: { text: string }) => (
+  <div className="toolTip">
+    <span className="toolTipText">{text}</span>
+  </div>
+);
+
+function NotificationContextMenu({
+  notification,
+  onMenuOptionClicked,
+}: {
+  notification: INotificationCardData;
+  onMenuOptionClicked: (
+    notificationId: string,
+    action: NotificationContextMenuAction
+  ) => void;
+}) {
+  return (
```

Comment:
This menu is good, nice touch with the tooltips, though I guess for mobile/touch screen users we could do with a conformation popup? Also for now if you wanted could put 'not yet implemented' in the confirm boxes

Reply:
This is just in response to a context menu action (but not the swipe, right?)